$NetBSD: patch-Makefile,v 1.1 2020/05/31 15:20:22 nia Exp $

Don't override things we want to pass through from pkgsrc cleanly.

--- Makefile.orig	2020-05-29 12:45:55.000000000 +0000
+++ Makefile
@@ -40,12 +40,8 @@ CC_AS    ?= ${CC_PREFIX}as
 
 MFLAGS   := 
 ASFLAGS  := 
-LDFLAGS  :=
 LDFLAGS_END :=
 INCFLAGS :=
-LIBS     :=
-CFLAGS   := 
-CXXFLAGS :=
 
 GIT_VERSION := " $(shell git rev-parse --short HEAD || echo unknown)"
 ifneq ($(GIT_VERSION)," unknown")
